### PR TITLE
Added sponsor info text below email input

### LINKED ISSUE #942: "Contact Us" button/link
We might have a dedicated button for this but not quite sure yet on how it would fit UX-wise.
One possibility is to have `interested in sponsoring? email sponsorship@coderit.org` on the site.

### PR DESCRIPTION
**Pull request by peterkos**
Fixes #942 

Adds a contact email for sponsors to get in touch w/ the BH team

image

Although this doesn't have the centering fixes in #938 it does have higher priority than that PR.

## Notes
- Only present on desktop; mobile layout needs to be refactored to be more pragmatic
- Mobile layout also needs to be tweaked to make room for this blurb w/o overflowing below the fold

### CODE REVIEW

File: src/preregister.js [hunk 1]
```diff
@@ -25,20 +25,25 @@ class PreRegister extends React.Component {
...
                         </div>
-                        
+
                         <img className="hero-img desktop-hide" src={HeroImage} alt="A decorative background hero graphic."/>
-            
+
                         <MailchimpInput/>
 
-                        <CovidPopover 
+                        <div id="contact">
```
Comment (ww2497) [inline src/preregister.js:37]: I would suggest adding 'mobile-hide' to its classes so this doesn't mess with spacing on mobile devices
Right now it has no size-scaling so it looks big on smaller devices & pushes things down (the mobile COVID notice is definitely off-screen for iPhone 5/6/7) 
Alternatively I can play around with vertical spacing if you think it's important to have this be visible on mobile

**Comment by ww2497** [comment]
Approved b/c of high priority but I highly suggest refining for mobile

**Comment by peterkos** [comment]
Bumped up the margin around the form: 10px->20px
image

**Comment by peterkos** [comment]
Will be addressed in #947

**Pull request merged**